CIP-0005/README.md

```diff
@@ -68,18 +68,19 @@ We define the following set of common prefixes with their corresponding semantic
 
 #### Hashes
 
-| Prefix             | Meaning                                            | Contents                                               |
-| ---                | ---                                                | ---                                                    |
-| `asset`            | Fingerprint of a native asset for human comparison | See [CIP-0014]                                         |
-| `pool`             | Pool operator verification key hash (pool ID)      | blake2b\_224 digest of an operator verification key    |
-| `script`           | Script hash                                        | blake2b\_224 digest of a serialized transaction script |
-| `addr_vkh`         | Address verification key hash                      | blake2b\_224 digest of a payment verification key      |
-| `addr_shared_vkh`  | Shared address verification key hash               | blake2b\_224 digest of a payment verification key      |
-| `stake_vkh`        | Stake address verification key hash                | blake2b\_224 digest of a delegation verification key   |
-| `stake_shared_vkh` | Shared stake address verification key hash         | blake2b\_224 digest of a delegation verification key   |
-| `vrf_vkh`          | VRF verification key hash                          | blake2b\_256 digest of a VRF verification key          |
-| `datum`            | Output datum hash                                  | blake2b\_256 digest of output datum                    |
-| `script_data`      | Script data hash                                   | blake2b\_256 digest of script data                     |
+| Prefix             | Meaning                                            | Contents                                                  |
+| ---                | ---                                                | ---                                                       |
+| `asset`            | Fingerprint of a native asset for human comparison | See [CIP-0014]                                            |
+| `pool`             | Pool operator verification key hash (pool ID)      | blake2b\_224 digest of an operator verification key       |
+| `script`           | Script hash                                        | blake2b\_224 digest of a serialized transaction script    |
+| `addr_vkh`         | Address verification key hash                      | blake2b\_224 digest of a payment verification key         |
+| `addr_shared_vkh`  | Shared address verification key hash               | blake2b\_224 digest of a payment verification key         |
+| `stake_vkh`        | Stake address verification key hash                | blake2b\_224 digest of a delegation verification key      |
+| `stake_shared_vkh` | Shared stake address verification key hash         | blake2b\_224 digest of a delegation verification key      |
+| `signer_vkh`       | Required signer verification key hash              | blake2b\_224 digest of a required signer verification key |
```

Comment:
After discussions on IOHK Slack we'd also like to add the `signer_vkh` prefix which is to be used to display the [`required_signers`](https://github.com/input-output-hk/cardano-ledger/blob/master/eras/alonzo/test-suite/cddl-files/alonzo.cddl) field of `tx_body`.

The existing prefixes are not suitable, because a HW wallet cannot determine whether the key is a payment (1852`) key, shared key (1854`) or a stake key so it wouldn't know which prefix to use. Another alternative would be to display the key hashes as hex, but that's also not ideal.

(the diff is showing also changes in the other prefixes but only the alignment of the `Contents` column has changed in those lines - GitHub UI doesn't show it too well)

Reply:
Do you not want to include "required" or "req" somewhere in that prefix? We could have other cases where we need a vhk from a signer, this is a bit ambiguous.

Reply:
Makes sense, thank you. Changed in https://github.com/cardano-foundation/CIPs/pull/177/commits/f991d4dee03f322225e950592fd9cbbf45bb15cf.